docs/make.jl

```diff
@@ -14,20 +14,31 @@ format = Documenter.HTML(prettyurls = get(ENV, "CI", nothing) == "true",
                          assets = assets)
 
 makedocs(
-    modules  = [ImageFiltering, OffsetArrays, Kernel, KernelFactors, ImageFiltering.MapWindow],
-    format   = format,
-    sitename = "ImageFiltering",
-    pages    = [
```

Comment:
Just a glance of the file changes:
1. It is recommended to preserve existing spaces for readability while adding new lines.
2. Please remember to add `.DS_Store` to the `.gitignore` file to avoid unnecessary commits.